src/pe-simple-header/widget/pe-simple-header.u-client-script.js

```diff
@@ -0,0 +1,233 @@
+function($rootScope, $scope, $window, $location,$anchorScroll, $interval, $timeout, $uibModal, $sce, snRecordWatcher, cabrillo, spUtil, spAriaUtil) {
+/* widget controller */
+	var c = this;
+	
+	try{
+		 $injector.get('spGtd');
+	}catch(e){
+		console.log('Prior to London Guided Tours did not exist.');
```

Comment:
- [ ] remove console.log

Reply:
This is part of a 'try/catch' - it will only log the information and it informative to 'why' the injector failed.  This is a standard practice to inform why something failed as part of a try/catch scenario when logging to the system provides zero value and the value is to the user not the system admins.

Reply:
your response is valid